apisix/script.lua

```diff
@@ -0,0 +1,40 @@
+local require    = require
+local core       = require("apisix.core")
+local pcall      = pcall
+local loadstring = loadstring
+
+local _M = {
+}
+
+
+function _M.load_script(route, api_ctx)
+	local script = route.value.script
+	if script == nil or script == "" then
+		return nil
+	end
+
+	local loadfun = loadstring(script)
```

Comment:
we should catch the return value, it may be failing. we need to record the error message.

Reply:
the second value is useful, eg: `"=route#id"` or `"=service#id"`